scripts/simplify_network.py

```diff
@@ -577,7 +740,7 @@ def consense(x):
     # treatment of outliers (nodes without a profile for considered carrier):
     # all nodes that have no profile of the given carrier are being aggregated to closest neighbor
     if (
-        snakemake.config.get("clustering", {})
+        snakemake.config.get("cluster_options", {})
```

Comment:
Before you already define the object cluster_config that is equivalent to this. May be worth using that variable

Reply:
`cluster_config` includes only `simplify_network` part:

https://github.com/ekatef/pypsa-earth/blob/e356862218e88b14783b07acae15ed9878955cc4/scripts/simplify_network.py#L725

While we are interested in the another part, namely `cluster_network`:

https://github.com/ekatef/pypsa-earth/blob/e356862218e88b14783b07acae15ed9878955cc4/config.default.yaml#L57-L68

Although, agree that linking to the config could be done in a more clear way...

Reply:
No no it's ok, I was just noting that few lines earlier, there is the definition cluster_options = snakemake.config.get("cluster_options", {}) that matches that line.
So we could use directly the dictionary cluster_options.

Reply:
I'm able to find only `cluster_config = snakemake.config["cluster_options"]["simplify_network"]` :( But that is not exaclty what we need...
Do I miss something?

Reply:
Right! rtm :)